### ISSUE TITLE
Teaching bubble close button no longer supports RTL

### ISSUE THREAD

**Issue by orokanasaru** [OfficeDev/office-ui-fabric-react#10302]
#### Actual behavior:

Currently looks like this in RTL:

![image](https://user-images.githubusercontent.com/5751627/63894085-a1528180-c9a0-11e9-906d-0fd36a7ee84f.png)

#### Expected behavior:

Just as it was after #453, it should look like:

![image](https://cloud.githubusercontent.com/assets/1434956/19732983/73c6d182-9b57-11e6-8668-513bcae46313.png)

### Priorities and help requested:

Are you willing to submit a PR to fix? No

Requested priority: Normal

**Comment by orokanasaru** [comment]
The header also shouldn't have `margin-right: 10px` in RTL.

**Comment by maxwellred** [comment]
Hi @orokanasaru Thanks for the issue! I'll take a look and get back to you soon.

**Comment by maxwellred** [comment]
Hi @orokanasaru Could you please let me know what specific version of Fabric you are on because i can't repro this in 7 or 6, thanks!

**Comment by orokanasaru** [comment]
This was done on 7.10 on our site, and whatever version is running on https://developer.microsoft.com/en-us/fabric. 

Further testing shows that this only happens when direction is changed to RTL after the package loads (though before the element is created/shown). 

This is probably not a common scenario outside of testing, so this can probably be changed to low priority or even closed outright.

**Comment by maxwellred** [comment]
@orokanasaru Ah alright, well I took a look further into how the locally spun up repo handles RTL and after you select RTL it fires location.reload() so maybe implementing page reload into your testing suite will grab the changes? Let me know if this helps and if this resolves your issue could you close the issue, thanks!
